src/debug/util/sorobanUtils.ts

```diff
@@ -431,45 +461,46 @@ const convertTupleToScVal = (tupleArray: TupleValue[]) => {
 };
 
 type PrimitiveArg = { type: string; value: unknown };
-type EnumArg = { tag: string; values?: unknown[] };
 
 const getScValFromPrimitive = (v: PrimitiveArg) => {
   if (v.type === "bool") {
-    const boolValue = v.value === "true";
+    const boolValue = v.value === "true" ? true : false;
```

Comment:
Do you gain anything from adding this ternary? I wouldn't block on asking it to be removed, but I'm curious.